src/components/CredderRating/CredderRating.js

```diff
@@ -10,7 +10,7 @@ import {fonts} from '../../utils/fonts';
 
 const CredderRating = ({containerStyle = {}, score}) => {
   const renderCredderRatingIcon = () => {
-    if (!score || score < 0)
+    if ((!score && score !== 0) || score < 0)
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The updated condition `if ((!score && score !== 0) || score < 0)` is a good improvement over the previous one. It now correctly handles the case where `score` is `0`, which is a valid value but would have been incorrectly treated as falsy in the old code.

```diff
-    if (!score || score < 0)
+    if ((!score && score !== 0) || score < 0)
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->